dbadmin/dbadmin.go

```diff
@@ -83,10 +83,23 @@ func Start() {
 
 	schemaMigration := utils.Cfg.SchemaMigration // Check env variable to migrate to specific version
 
-	if schemaMigration < 0 {
-		err = m.Up() // Upgrade to the latest
+	version, dirty, err := m.Version()
+	if err == migrate.ErrNilVersion {
+		log.Printf("No migration has been applied yet")
+	} else if err != nil {
+		log.Fatalf("Unexpected error occurred: %s", err.Error())
+	}
+	log.Printf("Current schema version: %d", version)
+
+	if dirty {
+		log.Printf("Forcing migration version")
+		err = m.Force(schemaMigration) // Force specified migration version if DB is dirty
```

Comment:
Lets add some additional check here, the default value is `-1` and the `Force` function will probably fail with that? For example force the version only when `schemaMigration > 0`, this means it'll not force anything until somebody comes and sets the specific version manually.